docs/user-guide/support-level.md

```diff
@@ -0,0 +1,105 @@
+- **tech-preview**: Feature is not fully supported, may not be functionally complete, and not suitable for
+  deployment in production. However, these features are provided to the customer as a courtesy and the primary goal
+  is for the feature to gain wider exposure with the goal of full support in the future.
+- **dev-preview**: Feature is considered for possible inclusion into future releases in order to collect feedback about the feature.
+
+The Support Level API has two different endpoints:
+- `GET /v2/support-levels/features` - Get the support level for a given OpenShift version and/or CPU architecture
+- `GET /v2/support-levels/architectures` - Get the supported CPU architectures for a given OpenShift version
+
+Note that the previous API `/v2/feature-support-levels` is now deprecated.
+
+
+## Feature Support Level
+This endpoint returns a list of supported features for a specific OpenShift version. 
+The support level can be evaluated by passing the CPU architecture as a query parameter.
+The result is evaluated dynamically. If a feature isn't supported on the given architecture, passing CPU architecture 
+will change only the support level and not the amount of features.
+
+
+
+### Using the Feature Support Level Endpoint
+
+To get a list of feature support levels on OpenShift version X, use the following command:
+
+```bash
+curl --request GET '<HOST>:<PORT>/api/assisted-install/v2/support-levels/features?openshift_version=<X>'
+```
+
+To get a list of feature support levels on OpenShift version X with Y CPU architecture, use the following command:
+
+
+```bash
+curl --request GET '<HOST>:<PORT>/api/assisted-install/v2/support-levels/features?openshift_version=<X>&cpu_architecture=<Y>'
+```
+
+Note that if CPU architecture is not passed as a parameter to the request, it will address the CPU architecture as x86_64.
+
+Response example for OpenShift version 4.12 with ARM CPU architecture:
+```json
+{
+  "features": {
+    "CLUSTER_MANAGED_NETWORKING": "supported",
+    "CNV": "unsupported",
+    "CUSTOM_MANIFEST": "supported",
+    "DUAL_STACK_VIPS": "unsupported",
+    "LVM": "dev-preview",
+    "NUTANIX_INTEGRATION": "unsupported",
+    "ODF": "unsupported",
+    "SINGLE_NODE_EXPANSION": "supported",
+    "SNO": "supported",
+    "USER_MANAGED_NETWORKING": "supported",
+    "VIP_AUTO_ALLOC": "dev-preview",
+    "VSPHERE_INTEGRATION": "supported"
+  }
+}
+```
+
+## Architecture Support Level
+This endpoint returns a list of supported architectures for a specific OpenShift version. 
+The service dynamically generates and returns a list of architectures and their support level for the given OpenShift version.
+
+
+### Using the Architecture Support Level Endpoint
+
+To get a list of architecture support levels on OpenShift version X, use the following command:
+
+```bash
+curl --request GET '<HOST>:<PORT>/api/assisted-install/v2/support-levels/architectures?openshift_version=<X>'
+```
```

Comment:
~~~suggestion
To get a list of architecture support levels on OpenShift version <version>, use the following command:

```bash
curl --request GET '<HOST>:<PORT>/api/assisted-install/v2/support-levels/architectures?openshift_version=<version>'
```
~~~